python/paddle/fluid/framework.py

```diff
@@ -691,6 +691,35 @@ def _set_error_clip(self, error_clip):
         """
         self.error_clip = error_clip
 
+    def _set_info(self, key, value):
+        """
+        Set some key-value information associated to this variable.
+
+        Args:
+            key(str): The key of the information entry.
+            value(object): The calue of the information entry.
```

Comment:
(typo) The **value** of the information entry.

Reply:
done